ovs-transfer-obj/src/main/java/org/dcsa/ovs/transferobjects/enums/PortCallStatusCode.java

```diff
@@ -14,4 +17,13 @@ public enum PortCallStatusCode {
   SLID("Sliding");
 
   private final String name;
+
+  public static Optional<PortCallStatusCode> valueFromString(String value) {
+    for (PortCallStatusCode pcsc : PortCallStatusCode.values()) {
+      if (Objects.equals(value, pcsc.toString())) {
+        return Optional.of(pcsc);
+      }
+    }
+    return Optional.empty();
```

Comment:
Isn't this "just" a slow version `<EnumClass>.valueOf()` (with a try-catch for `IllegalArgumentException` to return `Optional.empty()`)?